packages/map-template/src/components/Search/Search.jsx

```diff
@@ -316,14 +324,30 @@ function Search({ onSetSize, isOpen }) {
         }
     }, [useKeyboard]);
 
+    /*
+     * Get the legend sections and determine 
+     * If the legend button should be shown.
+     */
+    useEffect(() => {
+        const sections = getLegendSections(kioskLocation);
+        if (sections.length > 0) {
```

Comment:
This crashes the map template in case there are no Legend sections.

Reply:
Added check for existence of kiosk location